queries/ne.jinja2

```diff
@@ -195,8 +195,7 @@ SELECT
   NULL::jsonb AS __earth_properties__,
 
   jsonb_build_object(
-    'min_zoom', 4,
-    'area', way_area::bigint,
```

Comment:
Looks like you saw it in https://github.com/tilezen/vector-datasource/pull/1362#discussion_r131708319 :)